nestkernel/simulation_manager.cpp

```diff
@@ -430,6 +431,7 @@ nest::SimulationManager::prepare()
       * kernel().connection_manager.get_min_delay();
     kernel().music_manager.enter_runtime( tick );
   }
+  prepared_ = true;
```

Comment:
We should check at the top of the function that `! prepared_` and otherwise throw an exception like in the other checks.